### PR TITLE
update repository with NPWG git repos URLs

### PR DESCRIPTION
**Pull request by mcoquelin**
The code and documentation  was still pointing to Intel's
repos for both srio-network-device-plugin and srio-cni.
This patch changes to to NPWG's repos.

Signed-off-by: Maxime Coquelin <maxime.coquelin@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/intel/sriov-cni.svg?branch=master)](https://travis-ci.org/intel/sriov-cni) [![Go Report Card](https://goreportcard.com/badge/github.com/intel/sriov-cni)](https://goreportcard.com/report/github.com/intel/sriov-cni)
```
Comment (moshe010) [inline README.md:1]: same comment on changing to travis-ci.org/k8snetworkplumbingwg

Comment (mcoquelin) [reply README.md:1]: I will send a new revision fixing it. Thanks

**Review by moshe010 — APPROVED** [review]

**Review by adrianchiris — APPROVED** [review]

**Comment by martinkennelly** [comment]
@adrianchiris Are you happy to merge this?

**Comment by adrianchiris** [comment]
@martinkennelly  yes,  Mellanox CI issue not related to this PR.

**Pull request merged**